site/docs/README.md

```diff
@@ -84,5 +84,5 @@ Works! :tada:
 
 ---
 
-grammY supports the Telegram Bot API 5.5 which was [released](https://core.telegram.org/bots/api#december-7-2021) on December 7, 2021.
-(Last highlight: Responses to Join Requests)
+grammY supports the Telegram Bot API 5.6 which was [released](https://core.telegram.org/bots/api#december-30-2021) on December 30, 2021.
+(Last highlight: Spoiler and protected content support)
```

Comment:
```suggestion
(Last highlight: Spoiler and Improved Protected Content Support)
```

Reply:
This should be committed. Otherwise, I approve.